examples/map_transformations/autoalign_aia_hmi.py

```diff
@@ -0,0 +1,66 @@
+"""
+==============================
+Auto-Aligning AIA and HMI Data
+==============================
+
+This example shows how to auto-align two images with different reference frames.
+
+Here we use the optional keyword ``autoalign`` when calling Map's
+:meth:`~sunpy.map.GenericMap.plot` method.  See
+:ref:`sphx_glr_generated_gallery_map_transformations_reprojection_align_aia_hmi.py`
+for an alternate approach to image alignment.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+######################################################################
+# We use the AIA image and HMI image from the sample data.  For the
+# HMI map, we use the special HMI color map, which expects the plotted
+# range to be -1500 to 1500.
+
+map_aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+map_hmi = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
+map_hmi.plot_settings['cmap'] = "hmimag"
+map_hmi.plot_settings['norm'] = plt.Normalize(-1500, 1500)
+
+######################################################################
+# Plot both images side by side.  Note that the HMI image is oriented
+# "upside down" relative to the AIA image.
+
+fig = plt.figure(figsize=(12, 5))
+ax1 = fig.add_subplot(1, 2, 1, projection=map_aia)
+map_aia.plot(axes=ax1, clip_interval=(1, 99.9)*u.percent)
+ax2 = fig.add_subplot(1, 2, 2, projection=map_hmi)
+map_hmi.plot(axes=ax2)
+
+######################################################################
+# Set ``autoalign=True`` allows plotting the HMI image onto axes
+# defined by the AIA reference frame.  We also need to manually set
+# the plot limits because Matplotlib gets confused by the off-disk
+# parts of the image.  Note that the HMI image now has the same
+# orientation as the AIA image.
+
+fig = plt.figure(figsize=(12, 5))
+ax1 = fig.add_subplot(1, 2, 1, projection=map_aia)
+map_aia.plot(axes=ax1, clip_interval=(1, 99.9)*u.percent)
+ax2 = fig.add_subplot(1, 2, 2, projection=map_aia)
+map_hmi.plot(axes=ax2, autoalign=True, title='HMI image in AIA reference frame')
+ax2.dataLim = ax1.dataLim
+
+######################################################################
+# We can directly plot them over one another, by setting the
+# transparency of the HMI plot.
+
+fig = plt.figure()
+ax1 = fig.add_subplot(1, 1, 1, projection=map_aia)
+map_aia.plot(axes=ax1, clip_interval=(1, 99.9)*u.percent)
+map_hmi.plot(axes=ax1, autoalign=True, alpha=0.5)
+plt.title('HMI overlaid on AIA')
```

Comment:
Rather than cracking and using plt right at the end? :grin: 
```suggestion
ax1.set_title('HMI overlaid on AIA')
```

Reply:
Done